test/openAPI/features/list_clients.feature

```diff
@@ -1,13 +1,29 @@
+@method=GET @endpoint=/listClients
 Feature: This endpoint is used to retrieve the list of clients from GovStack.
 
-  Request endpoint: GET /listClients
-
+  @smoke @unit @positive
   Scenario: Successfully retrieved the list of clients from GovStack
-    Given Wants to retrieve the the list of Clients of GovStack
-    When The request to retrieve the list of Clients of GovStack is sent
-    Then The operation returns the list of Clients of GovStack
 
+    Given User wants to retrieve the the list of Clients of GovStack
+    When User sends GET request
+    Then User receives a response from the listClients endpoint
+    And The listClients endpoint response should be returned in a timely manner
+    And The listClients endpoint response should have status 200
+    And The listClients endpoint response should match json schema
+
+  @unit @positive
   Scenario: Successfully retrieved the list of clients from GovStack with optional parameters in the request
-    Given Wants to retrieve the list of clients from GovStack with optional parameters specified
-    When The request with optional parameters to retrieve the list of Clients of GovStack is sent
-    Then The operation returns the list of Clients of GovStack
+    Given User wants to retrieve the the list of Clients of GovStack
+    When User sends GET request
+    And User provides optional query parameters "<serviceId>" as serviceId, "<instanceId>" as instanceId and optional header "<X-GovStack-Client>" as X-GovStack-Client
```

Comment:
listClients have only one optional parameter <instanceId>.
`X-GovStack-Client` header is not needed. `Accept: application/json` header is needed.

Reply:
Okay, thank you @aleksander-reitsakas ! I'm going to fix it